packages/fire-emblem-heroes-stats/lib/stats.json

```diff
@@ -1,4 +1,243 @@
 {
+  "grand_battle_heroes": [
+    {
+      "name": "Flier",
+      "imageName": "Lance Flier",
```

Comment:
I'm thinking that I like the idea of a `shortName` rather than `imageName` on the hero.

The semantics seem slightly more correct.  The only reason you're diverging the two keys is so that you can consume the abbreviated version in spaces in the UI where a longer name wouldn't fit.  It's a bit weird if the only full, canonical name is in the `imageName` key.